README.md

```diff
@@ -8,91 +8,107 @@ iOS Playbook 📚
-| David Rodrigues         | [@dmcrodrigues](https://twitter.com/dmcrodrigues)             |
-| Diego Petrucci          | [@diegopetrucci](https://twitter.com/diegopetrucci)           |
-| Emese Toth              | [@emeseuk](https://github.com/emeseuk)                        |
-| Giorgos Tsiapaliokas    | [@gtsiap](https://github.com/gtsiap)                          |
-| Ilya Puchka             | [@ilyapuchka](https://twitter.com/ilyapuchka)                 |
-| James Birtwell          | [@jimmybee](https://github.com/jimmybee)                      |
-| Jason Dobo              | [@jasondobo](https://github.com/jasondobo)                    |
-| João Pereira            | [@NSMyself](https://twitter.com/nsmyself)                     |
-| Joshua Simmons          | [@j531](https://github.com/j531)                              |
-| Julien Ducret           | [@brocoo](https://github.com/brocoo)                          |
-| Konrad Muchowicz        | [@konrad-em](https://github.com/konrad-em)                    |
-| Martin Nygren           |                                                               |
-| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
-| Michał Kwiecień         | [@kwiecien_co](https://twitter.com/kwiecien_co)               |
-| Nicola Di Pol           | [@nicola.dipol](https://github.com/horothesun)                |
-| Olivier Halligon        | [@AliSoftware](https://twitter.com/aligatr)                   |
-| Rui Peres               | [@peres](https://twitter.com/peres)                           |
-| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
-| Simon Cass              | [@codercass](https://twitter.com/codercass)                   |
-| Jakub Tomanik           | [@jtomanik](https://github.com/jtomanik)                      |
-| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
-| Witold Skibniewski      |                                                               |
-| Yasuhiro Inami          | [@inamiy](https://twitter.com/inamiy)                         |
-| Yuri Karabatov          | [@karabatov](https://twitter.com/karabatov)                   |
-| Greg Bryant             |                                                               |
-| Sam Francis             |                                                               |
-| Patrick Westmeyer       |                                                               |
-| Daniel Spindelbauer     | [@sdaniel55](https://twitter.com/sdaniel55)                   |
+We're organised in Squads. Each squad can be composed of people from Engineering (iOS, Android, Web, Backend), Design and Product.
 
+Some of the roles are transverse to all the squads:
 
-## 2. Which squad each individual belongs to, and its availability.
+<!-- begin:roles -->
+<!--
+  DO NOT EDIT MANUALLY: This table has been auto-generated.
```

Comment:
Maybe we can add a pre-commit hook or something?

Reply:
It would be nice. Problem is, commit hooks are local, they are not committed and shared across the team. So each person in the team needs to install them on their machine individually

Reply:
Thanks for the suggestion!
I will investigate this and see if it is possible to trigger it only for a change in a specific file.

Reply:
If pre-commit hooks are indeed local, the only way to check would be to add some CI. But adding CircleCI on the playbook would mean add more stuff to the CircleCI queue which is already quite under pressure; Maybe it would be the occasion to try GitHub Actions (especially since, for the playbook, we could use Linux containers, no need for macOS VMs) to run Danger on it? 🤔

Reply:
> commit hooks are local
😕 I wasn't aware of this.

Reply:
Well it might still be an idea to investigate as there are suggestions to share hooks accross the team: https://www.viget.com/articles/two-ways-to-share-git-hooks-with-your-team/

But that still require all members in the team to update their configuration. Which we could imagine automate via the `ios-onboarding` bash script we already have (see our [New Hires Checklist](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md#setup-guide)) – and ask existing team members to update their config manually once and for all too

Reply:
I just foresee everyone forgetting to update the generated files %) Yeah, not very good that commit hooks are local.

Playbook is open source though so we could run e.g. free Travis on it, no?

Reply:
I just realize that even if we share git hooks across teams (using the onboarding script to install on everyone's laptop for example) it still won't perform any check for changes made directly via GitHub suggestions for example. So it's not probably an option anymore even if we managed to sync them on everybody's machine.

yeah CI is the best option. CircleCI has free plans for OSS repos too (that's where all my OSS repos are) so maybe that's the best route to go? I'd prefer CircleCI (to keep what we already use elsewhere and not have to learn 2 different syntaxes depending on repos) or GitHub Actions (since it's new and would be a nice occasion to test it and see if it could be an option even for the main repo in the future) than travis-ci tbh

Reply:
I only said Travis because I know 100% it supports open source repos for free. Sure it's better to use either CircleCI or try out Actions.

This is just a use case that's begging for CI.